### PR TITLE
Integrates Josh's Cadence style guide.

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: app/data/doc-collections/cadence__styleguide.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "source": {
+      "owner": "onflow",
+      "name": "cadence-style-guide",
+      "branch": "main",
+      "rootPath": "docs/"
+    }
+}
```
Comment (10thfloor) [inline app/data/doc-collections/cadence__styleguide.json:8]: Register the repo with the docs site backend.

File: app/data/doc-collections/index.ts [hunk 2]
```diff
@@ -45,6 +46,7 @@ export const testCollections = {
 export const docCollections = {
   ...(includeTestingDocCollections ? testCollections : {}),
   cadence,
+  "cadence/style-guide": cadence__styleguide,
```
Comment (10thfloor) [inline app/data/doc-collections/index.ts:49]: Specify the base URL for the repo's `docs` folder content. 
In this case the files in your repo will appear under `/cadence/style-guide/your-doc`

File: app/data/doc-collections/cadence.json [hunk 3]
```diff
@@ -6,231 +6,6 @@
     "rootPath": "docs/"
   },
   "manifest": {
-    "displayName": "Cadence",
```
Comment (10thfloor) [inline app/data/doc-collections/cadence.json:9]: Ignore this.

File: app/data/doc-collections/cadence__styleguide.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "source": {
+    "owner": "onflow",
+    "name": "cadence-style-guide",
+    "branch": "main",
+    "rootPath": "docs/"
+  },
+  "manifest": {
```
Comment (10thfloor) [inline app/data/doc-collections/cadence__styleguide.json:8]: Your flow-docs.json is the "manifest" for this doc collection. 
A display name is required here. A user-defined display name will overwrite this if present.

**Review by joshuahannan — APPROVED** [review]
I'm not super familiar with how it all fits together, but it seems ok to me! Someone else might need to take a look first though

**Review by bthaile — APPROVED** [review]
approving, we'll see how Cadence style guide turns out.

**Pull request merged**